src/main/java/inventorysetups/InventorySetupsConfig.java

```diff
@@ -365,11 +365,23 @@ default InventorySetupsShowWornItemsFilterID showWornItemsFilter()
 		return InventorySetupsShowWornItemsFilterID.All;
 	}
 
+	@ConfigItem(
+			keyName = CONFIG_KEY_SECTION_MODE,
```

Comment:
Don't use the same config option as section mode, this will overwrite data in the settings.properties. Use a new config key